schema/bundle.schema.json

```diff
@@ -47,23 +47,18 @@
           "description": "The type of image. If this is not specified, 'oci' is assumed",
           "type": "string"
         },
+        "labels": {
+          "description": "Key/value pairs that used to specify identifying attributes of images",
+          "type": "object",
+          "additionalProperties": {"type": "string"}
+        },
         "mediaType": {
           "description": "The media type of the image",
           "type": "string"
         },
-        "platform": {
-          "description": "The target platform",
-          "properties": {
-            "architecture": {
-              "description": "The architecture of the image (i386, amd64, arm32, arm64,...)",
-              "type": "string"
-            },
-            "os": {
-              "description": "The operating system of the image (linux, windows, darwin,...)",
-              "type": "string"
-            }
-          },
-          "type": "object"
+        "originalImage": {
```

Comment:
Was this part missing in the spec? Do you think you could add it in another PR? Or at least in a separate commit, just for a separation of concern reason ?

Reply:
Ok just dug it a little, @glyn removed `originalImage` field in this PR #218 
I guess that's a rebase artifact 😅

Reply:
Correct. `originalImage` is no more and should be deleted from this PR.

Reply:
Oops! yeah, rebase goof. Fixing.